src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -369,7 +395,8 @@
                 return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
                     && callback.getEvent() == Event.SUBMIT_APPEAL
```

Comment:
Does this need to include check for PAY_AND_SUBMIT_APPEAL

Reply:
It is not required.